sklearn/cluster/optics_.py

```diff
@@ -168,18 +168,23 @@ class OPTICS(BaseEstimator, ClusterMixin):
 
     Parameters
     ----------
-    min_samples : int
+    NOTE: should we call this parameter min_neighbors, or n_neighbors,
+    to be more consistent with the NearestNeighbors API, which this parameter
+    is used for?
+    min_samples : int (default=5)
         The number of samples in a neighborhood for a point to be considered
         as a core point.
 
-    max_bound : float, optional
+    NOTE: max_radius (similar to NearestNeighbors API)
+    or max_eps (to follow paper's jargon)
+    max_bound : float, optional (default=np.inf)
```

Comment:
I believe this parameter I believe is understood better as `max_eps` from the algorithm and DBSCAN perspective.

Reply:
I'm happy with max_eps. I thought I responded to this elsewhere.